proxy/api/src/http/project.rs

```diff
@@ -137,22 +161,32 @@ mod handler {
         Ok(reply::json(&project::get(&state, &urn)?))
     }
 
-    /// List all known projects.
-    ///
-    /// If [`super::ListUser::user`] is given we only return projects that this user tracks.
-    pub async fn list(
+    /// List all projects tracked by the current user.
+    pub async fn list_tracked(ctx: context::Ctx) -> Result<impl Reply, Rejection> {
+        let ctx = ctx.read().await;
+        let state = ctx.state.lock().await;
+        let projects = project::Projects::list(&state)?.tracked;
+
+        Ok(reply::json(&projects))
+    }
+
+    /// List all projects the current user has contributed to.
+    pub async fn list_contributed(ctx: context::Ctx) -> Result<impl Reply, Rejection> {
+        let ctx = ctx.read().await;
+        let state = ctx.state.lock().await;
+        let projects = project::Projects::list(&state)?;
+
+        Ok(reply::json(&projects.contributed))
+    }
+
+    /// List all known projects tracked by user of given `user_id`
+    pub async fn list_for_user(
```

Comment:
I find this confusing. The fn is called `list_for_user` which doesn't provide enough info, and reading the docs I see it lists projects tracked by the given `user_id`. So two suggestions

1. Could we rename it to `list_tracked_by_user` or just `list_tracked_by`
   Not only makes it easier to understand but it's also consistent with the above `fn list_tracked`.

2. Rename the underlying used function, `list_projects_for_user`, the same way